v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -130,13 +130,24 @@ public interface KafkaToBigQueryFlexOptions
       parentName = "avroFormat",
       parentTriggerValues = {"CONFLUENT_WIRE_FORMAT"},
       optional = true,
-      description = "Schema Registry Connection URL.",
+      description = "Schema Registry Connection URL",
       helpText =
           "Schema Registry Connection URL for a registry which supports Confluent wire format.")
   String getSchemaRegistryConnectionUrl();
 
   void setSchemaRegistryConnectionUrl(String schemaRegistryConnectionUrl);
 
+  @TemplateParameter.Boolean(
+      order = 11,
+      optional = true,
+      description = "Persist the Kafka Message Key to the BigQuery table",
+      helpText =
+          "If true, the pipeline will persist the Kafka Message Key in the BigQuery table, in a `_key` field. Default is false (Key is ignored).")
```

Comment:
```suggestion
          "If true, the pipeline will persist the Kafka message key in the BigQuery table, in a `_key` field of type `bytes`. Default is false (Key is ignored).")
         
```
mention we will only write bytes directly

Reply:
Done